src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -433,6 +438,48 @@ class ComposterOverlay {
         return first ?: error("First is empty!")
     }
 
+    private fun getMaterials(internalName: String, itemName: String, itemsNeeded: Double) {
+        val sackData = fetchSackItem(internalName.asInternalName())
+        val amountInSacks = sackData.amount
+        val sacksLoaded = sackData.outdatedStatus
+        val having = InventoryUtils.countItemsInLowerInventory { it.getInternalName() == internalName.asInternalName() }
+
+        if (itemsNeeded.toInt() == 0 || itemName.removeColor() == "Biofuel") return
+
+        if (config.composterOverlayGetType == 0) {
+            inInventory = false
+            BazaarApi.searchForBazaarItem(itemName, itemsNeeded.toInt())
+            return
+        }
+
+        if (sacksLoaded == -1) {
+            val sackType =
+                if (internalName == "VOLTA" || internalName == "OIL_BARREL") "Mining" else "Enchanted Agronomy"
+            clickableChat(
+                "§e[SkyHanni] Sacks could not be loaded. Click here and open your §9$sackType Sack §eto update the data!",
+                "sax"
+            )
+            return
+        } else if (amountInSacks == 0 && having == 0) {
```

Comment:
Maybe i'm not understanding this one but I'm not sure why the amount of items in the inventory would make a difference. If there's none in the sacks and less than the amount needed in the inventory it should go to bz for that amount remaining

Reply:
Uhmmmm. Ig my thinking was that if you have some in your inventory, you might not want to open the bz?
There's a clickableChat msg that opens the bz later

I might change this